utils/threadpool/threadpool.cpp

```diff
@@ -89,6 +89,7 @@ void ThreadPool::setQueueSize(size_t queueSize)
 void ThreadPool::pruneThread()
 {
     boost::unique_lock<boost::mutex> lock2(fPruneMutex);
+    utils::setThreadName("pruneThread");
```

Comment:
We don't need to extend crit section so setThreadName should be put in front of the mutex lock.

Reply:
I don't see as it makes much difference ... but done.